test/server.rollbar.test.js

```diff
@@ -116,6 +142,24 @@ vows.describe('rollbar')
         'should set configured options': function(r) {
           assert.equal('fake-env', r.options._configuredOptions.environment);
         }
+      },
+      'with valid tracer': {
```

Comment:
again not sure if we need a 'without tracer' test

Reply:
I agree with @austenLacy . This is covered by the invalid tracer test.